tests/cupy_tests/manipulation_tests/test_kind.py

```diff
@@ -10,6 +10,19 @@
 @testing.gpu
 class TestKind(unittest.TestCase):
 
+    @testing.for_all_dtypes()
+    def test_asarray_chkfinite(self, dtype):
```

Comment:
There are testing utilities that can help compare returned NumPy & CuPy arrays, parametrize array orders, etc. It looks like this file was rarely touched so it didn't get updated to use those, but it's never too late to modernize it 🙂
```suggestion
    @testing.for_orders('CFAK')
    @testing.for_all_dtypes()
    @testing.numpy_cupy_array_equal()
    def test_asarray_chkfinite(self, xp, dtype, order):
```

Reply:
> t looks like this file was rarely touched so it didn't get updated to use those, but it's never too late to modernize it :slightly_smiling_face: 

I totally agree with you and it seems Interesting. Shall I update other parts too? 
In this file, I think the following test cases can be updated:
- [test_asfortranarray](https://github.com/cupy/cupy/blob/0757aa31a10785e6b728ad465724e97a83945b73/tests/cupy_tests/manipulation_tests/test_kind.py#L21): All asfortranarray tests.
- [test_require_C_and_F_flags](https://github.com/cupy/cupy/blob/0757aa31a10785e6b728ad465724e97a83945b73/tests/cupy_tests/manipulation_tests/test_kind.py#L88)

Reply:
Feel free to work on it! But I'd suggest to create a separate PR so that each PR's scope is well defined.

Reply:
Yeah, sure. Thanks!!